ui/app/components/auth-form.js

```diff
@@ -276,6 +277,10 @@ export default Component.extend(DEFAULTS, {
       if (this.customPath || backend.id) {
         data.path = this.customPath || backend.id;
       }
+      // add nonce field for okta backend
+      if (backend.id == 'okta') {
+        data.nonce = uuidv4();
```

Comment:
Since we have discontinued support for IE 11 I wonder if we need to add the `uuid` library as a dependency or if [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) would do the trick?

Reply:
I removed the `uuid` library and used `crypto.randomUUID()` instead to generate the uuid values. Thank you for the suggestion!